graphql/resolvers.go

```diff
@@ -2419,10 +2428,17 @@ func (r *taskResolver) Annotation(ctx context.Context, obj *restModel.APITask) (
 		return nil, nil
 	}
 	apiAnnotation := restModel.APITaskAnnotationBuildFromService(*annotation)
+	return apiAnnotation, catcher.Resolve()
```

Comment:
I can't see all your commits (fyi, I would discourage too much force-pushing unless you're fixing a specific mistake; it makes it harder for the reviewer to follow changes, and it means we can't use the "see changes since your last review" option), but I thought you'd added this "getJiraTickets" thing directly to this Annotation query.  Does the thing you added to gqlgen.yml handle that?

Reply:
discussed over slack